src/libutil/simd_test.cpp

```diff
@@ -607,7 +607,7 @@ void test_arithmetic ()
 template<typename VEC>
 void test_fused ()
 {
-    typedef typename VEC::value_t ELEM;
+    // typedef typename VEC::value_t ELEM;
```

Comment:
These are unnecessary. You can feel free to delete those unused typedef lines entirely.